diff --git a/luci-app-ssr-plus/luasrc/model/cbi/shadowsocksr/servers.lua b/luci-app-ssr-plus/luasrc/model/cbi/shadowsocksr/servers.lua
--- a/luci-app-ssr-plus/luasrc/model/cbi/shadowsocksr/servers.lua
+++ b/luci-app-ssr-plus/luasrc/model/cbi/shadowsocksr/servers.lua
@@ -611,1 +611,0 @@
-s.description = translate("Node order can be dragged with the mouse and takes effect immediately. The automatic switch order of server nodes is consistent with the node order in the table.")
diff --git a/luci-app-ssr-plus/luasrc/view/shadowsocksr/server_table.htm b/luci-app-ssr-plus/luasrc/view/shadowsocksr/server_table.htm
--- a/luci-app-ssr-plus/luasrc/view/shadowsocksr/server_table.htm
+++ b/luci-app-ssr-plus/luasrc/view/shadowsocksr/server_table.htm
@@ -176,0 +177,1 @@
+		display: none !important;
